.evergreen/run-serverless-tests.sh

```diff
@@ -21,14 +21,11 @@ echo "Running serverless tests with ${JDK}"
 
 export JAVA_HOME="/opt/java/jdk11"
 
-MONGODB_URI_SINGLE_HOST=${MONGODB_URI%%,*}
-SINGLE_HOST=${MONGODB_URI_SINGLE_HOST:10}
-
-MONGODB_URI_FINAL="mongodb://${SERVERLESS_ATLAS_USER}:${SERVERLESS_ATLAS_PASSWORD}@${SINGLE_HOST}/?tls=true"
-
-echo ${MONGODB_URI_FINAL}
+# assume "mongodb" protocol for single server, and "mongodb+srv" protocol for multi server
```

Comment:
This is a bit of a hack, but it allows me not to have to write too much bash string manipulation code, which I despise.

Reply:
Ack - and agree.